src/module/combat/combat.js

```diff
@@ -566,8 +570,13 @@ export class CombatSFRPG extends Combat {
         const parts = [];
 
         if (this.getCombatType() === "starship") {
-            parts.push("@pilot.skills.pil.mod");
             rollContext.setMainContext("pilot");
+            if (combatant.actor.data.data.crew.useNPCCrew) {//If an NPC starship, don't apply the static bonus to Piloting from maneuverability/thrusters, as that's included in their piloting bonus
+                parts.push("@pilot.skills.pil.mod", "@combatant.attributes.systems.engines.mod", "@combatant.attributes.systems.powerCore.modOther", "@additional.modifiers.bonus");
+            } else {//Initiative rolls *probably* include system damage
+                parts.push("@pilot.skills.pil.mod", "@combatant.attributes.pilotingBonus.value", "@combatant.attributes.systems.engines.mod", "@combatant.attributes.systems.powerCore.modOther", "@additional.modifiers.bonus")
+            }
+                
```

Comment:
I created a separate issue for it. I think they are not related to each other and should be addressed separately.

Reply:
#707 is gonna have to be done separately because of how starship actions are stored and executed. That code there isn't addressing #707, that's #706. (Note that I edited both that function and the one above. That function is the one that has an affect, the one above seems deprecated but I edited it anyway)